planb/cli.py

```diff
@@ -97,6 +97,7 @@ def create(regions: list,
 @click.option('--ring-size', type=int, required=True)
 @click.option('--dc-suffix', default='', type=str, callback=validate_dc_suffix)
 @click.option('--num-tokens', default=256, type=int, help='number of virtual nodes per node, default: 256')
+@click.option('--allocate-tokens-for-keyspace', type=str, help='allocate tokens for specified keyspace on non-seed nodes')
```

Comment:
Shouldn't we add a reference in the docs as well?

Reply:
Good point, thanks.